_includes/sponsor_area.html

```diff
@@ -0,0 +1,46 @@
+<div class="team-area bg-color1">
+  <div class="team-wrapper section-spacing pb-0">
+    <div class="container">
+      <div class="section-title wow fadeInUp" data-wow-duration="2s">
+        <h2 class="heading-2">Behind HospitalRun</h2>
```

Comment:
On this page add a new section labeled "Emeritus" and put Project founders there. We need to be sure that if someone wants to contact a core member he knows who to contact.